common/lib/api-client/transformers/person.transformer.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function personTransformer(data) {
+  return {
+    ...data,
+    image_url: `/person/${data.id}/image`,
+    fullname: [data.last_name, data.first_names].filter(Boolean).join(', '),
```

Comment:
This is the code from `common/services/person` extracted out into a separate transformer

:+1:

Reply:
Yep. The benefit here is that it will be called **every** time devour deserializers it. Previously if something like a person resource was nested in a relationship (`move` -> `profile` -> `person`), we would have to do that manually for the move and person even if neither of these required any kind of transformation themselves.

So this is more consistent and hopefully harder to miss.